README.md

```diff
@@ -40,7 +40,7 @@ create a new key under "_Credentials_".
 
 `HUBOT_CYCLE_TIME_FMT` - Optionally ovveride the display format for times (see Moment-timezone.js). Default is "ddd hA" (e.g. "Sun 10pm")
 
-`HUBOT_CYCLE_TZ_NAME` - Optionally set the timezone name (e.g. 'EST' see Moment-timezone.js). This wins when set.
+`HUBOT_CYCLE_TZ_NAME` - Optionally set the timezone name (e.g. 'America/Chicago' see Moment-timezone.js). This wins when set.
```

Comment:
Can we clarify the verbiage here? What does "this wins" mean? What exactly happens when this env var is set? Does it override any `hubot cycle set offsetname` command?